services/forum/src/main.py

```
@@ -71,4 +138,24 @@ def create_comment(
+    post_id: int,
+    action: schemas.VoteAction,
+    db: Session = Depends(get_db),
+    user_id: int = Depends(get_current_user_id)
+):
+    return logic.cast_post_vote(db, post_id, user_id, action.vote_value)
+
+@router.post("/posts/{post_id}/comments/{comment_id}/vote", status_code=status.HTTP_200_OK)
+def vote_on_comment(
+    comment_id: int,
+    action: schemas.VoteAction,
+    db: Session = Depends(get_db),
```

Ruff (0.15.5)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)